docs/guide/data_types/maps.rst

```diff
@@ -379,56 +206,42 @@ dictionary. In the following example we change the title of the plot by changing
     import sunpy.data.sample
     import matplotlib.pyplot as plt
     smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    smap.plot_settings['title'] = "My Second sunpy Plot"
-    smap.plot_settings['cmap'] = plt.cm.Blues_r
     fig = plt.figure()
-    smap.plot()
+    smap.plot(cmap=plt.cm.Greys_r)
     plt.colorbar()
     plt.show()
 
+You can view or make changes to the default settings through the ``sunpy.map.GenericMap.plot_settings`` dictionary.
+See `Editing the colormap and normalization of a Map <https://docs.sunpy.org/en/stable/generated/gallery/plotting/map_editcolormap.html>`_ for an example of this method.
+
 
-Colormaps and Normalization
----------------------------
+4.2 Colormaps and Normalization
+-------------------------------
 
-Image data is generally shown in false color in order to better identify it or
-to better visualize structures in the image. Matplotlib handles this colormapping
-process through the `~matplotlib.colors` module. This process involves two steps:
-the data array is first mapped onto the range 0-1 using an instance of
-`~matplotlib.colors.Normalize` or a subclass; then this number is mapped to a
-color using an instance of a subclass of a `~matplotlib.colors.Colormap`.
+Image data is generally shown in false color in order to better identify it or to better visualize structures in the image.
+Matplotlib handles this colormapping process through the `~matplotlib.colors` module.
+First, the data array is mapped onto the range 0-1 using an instance of `~matplotlib.colors.Normalize` or a subclass.
+Then, the data is mapped to a color using a `~matplotlib.colors.Colormap`.
```

Comment:
I agree. It was for this reason that I linked to five "foundational" examples at the beginning of the Plotting Maps section. Since those examples cover many of the basics for plotting in short self-contained chunks, I see these plotting subsections as additional information for more advanced users.